build/Targets/VSL.Versions.targets

```diff
@@ -88,8 +88,8 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <CommonNuGetMoniker>beta2</CommonNuGetMoniker>
-    <VSSemanticVersion>15.3</VSSemanticVersion>    
+    <CommonNuGetMoniker>beta1</CommonNuGetMoniker>
+    <VSSemanticVersion>15.6</VSSemanticVersion>    
```

Comment:
If you do plan to ship something for 15.4 then I guess this probably needs to be flipped to 15.4 after this merge.